Lib/mailcap.py

```diff
@@ -170,7 +172,7 @@ def findmatch(caps, MIMEtype, key='view', filename="/dev/null", plist=[]):
     for e in entries:
         if 'test' in e:
             test = subst(e['test'], filename, plist)
```

Comment:
```suggestion
            test = subst(e['test'], MIMEtype, filename, plist)
```
Don't know that we should fix this, but this is actually broken if `test` has a `%s` in it.